lib/Serialization/SerializeSIL.cpp

```diff
@@ -392,10 +392,7 @@ void SILSerializer::writeSILFunction(const SILFunction &F, bool DeclOnly) {
   // Assign a value ID to each SILInstruction that has value and to each basic
   // block argument.
   unsigned ValueID = 0;
-  llvm::ReversePostOrderTraversal<SILFunction *> RPOT(
-      const_cast<SILFunction *>(&F));
-  for (auto Iter = RPOT.begin(), E = RPOT.end(); Iter != E; ++Iter) {
-    auto &BB = **Iter;
+  for (const SILBasicBlock &BB : F) {
```

Comment:
This is now the same code and style that existed in this file for ever. It uses it in other places as well. I don't want to change the style as part of this commit.